backend/layers/api/router.py

```diff
@@ -23,7 +23,7 @@ def portal_api():
     global api
     if api is None:
         database_provider = DatabaseProvider()
-        crossref_provider = CrossrefProviderInterface()
+        crossref_provider = CrossrefProvider()
```

Comment:
I can't remember if this works in rdev, since I originally set this up to only work in a deployed environment. The class wasn't changed at all though.